src/contracts/core/DelegationManager.sol

```diff
@@ -311,93 +327,87 @@ contract DelegationManager is
      * @notice Used to complete the specified `withdrawal`. The caller must match `withdrawal.withdrawer`
      * @param withdrawal The Withdrawal to complete.
      * @param tokens Array in which the i-th entry specifies the `token` input to the 'withdraw' function of the i-th Strategy in the `withdrawal.strategies` array.
-     * This input can be provided with zero length if `receiveAsTokens` is set to 'false' (since in that case, this input will be unused)
-     * @param middlewareTimesIndex is the index in the operator that the staker who triggered the withdrawal was delegated to's middleware times array
      * @param receiveAsTokens If true, the shares specified in the withdrawal will be withdrawn from the specified strategies themselves
      * and sent to the caller, through calls to `withdrawal.strategies[i].withdraw`. If false, then the shares in the specified strategies
      * will simply be transferred to the caller directly.
-     * @dev middlewareTimesIndex is unused, but will be used in the Slasher eventually
      * @dev beaconChainETHStrategy shares are non-transferrable, so if `receiveAsTokens = false` and `withdrawal.withdrawer != withdrawal.staker`, note that
      * any beaconChainETHStrategy shares in the `withdrawal` will be _returned to the staker_, rather than transferred to the withdrawer, unlike shares in
      * any other strategies, which will be transferred to the withdrawer.
      */
     function completeQueuedWithdrawal(
         Withdrawal calldata withdrawal,
         IERC20[] calldata tokens,
-        uint256 middlewareTimesIndex,
```

Comment:
we should have never had this, but we can't break this interface

Reply:
We absolutely can't break this? Is the assumption for our TUPS contracts is that all interfaces are final?
I would deem that for a smart contract system that is not fully implemented (missing rewards/slashing) its pretty reasonable to expect interfaces to change. But I guess it's not much benefit for a lot of downstream impact.

Reply:
Slashing is a breaking change, feel like this is fine? Worst case, add the new version and say we will deprecate the `middlewareTimesIndex` post slashing.

Reply:
slashing is not a breaking change to deposits/withdrawals if we don't want it to be

Reply:
Let's break the interface. Fuck this param, it's time

Reply:
> slashing is not a breaking change to deposits/withdrawals if we don't want it to be

i'd like it to be so we can kill this param

Reply:
If we don't kill middlewaretimes at this moment of time we never will.